types/src/transaction/authenticator.rs

```diff
@@ -578,12 +587,20 @@ impl AccountAuthenticator {
             Self::MultiEd25519 { signature, .. } => signature.to_bytes().to_vec(),
             Self::SingleKey { authenticator } => authenticator.signature_bytes(),
             Self::MultiKey { authenticator } => authenticator.signature_bytes(),
+            Self::NoAccountAuthenticator => vec![], // TODO: check this.
```

Comment:
todo?

Reply:
`NoAccountAuthenticator` contains neither a public key nor a signature. So, an empty vector is returned. I removed the TODOs.